controlpanel/api/models/user.py

```diff
@@ -84,9 +84,14 @@ def reset_mfa(self):
 
     def save(self, *args, **kwargs):
         try:
-            User.objects.get(pk=self.pk)
+            existing = User.objects.get(pk=self.pk)
+
         except User.DoesNotExist:
             cluster.User(self).create()
+            if self.is_superuser:
+                slack.notify_team(
+                    f"`{self.username}` was created as a superuser"
```

Comment:
Do we get a double notification when a user is created as superuser? (is it even possible to do so? I thought users created themselves when they logged in and then someone else could make them superuser if required)